### DISCUSSION TITLE
该项目是否有明确计划长期迭代?

### DISCUSSION THREAD

**Discussion by yoyo837** [eleme/morjs#80] [Q&A] [answered]
是否KPI项目? 无恶意, 纯粹担心没几天就不维护了, 如果不维护, 是否愿意交给社区志愿者之类的?

**Comment by lyfeyaj** [comment, marked as answer]
先说结论：**`MorJS` 不是一个 KPI 项目，是饿了么内部面向多端研发的首选及主要框架，被大量的项目所使用，会长期且持续的迭代和升级。**

早在 18 年，饿了么内部就有了基于小程序 DSL 的多端研发框架，也是 MorJS 的早期雏形，后续历经数年和若干个大版本的迭代与更新，以及上百个项目的验证和洗礼之后才正式对外开源并最终与大家见面。

MorJS 的诞生起源于饿了么在小程序多端研发方向上的朴素实践，我们不追求通过 `React` 或 `Vue` 来统一一切，也对重复造轮子没有兴趣，更不想卷入 DSL 的优劣之争，而是始终从业务诉求、灵活性、用户体验与性能出发，追求渐进增强和无感升级，这也是我们选择小程序 DSL 来实现多端的原因。

或许未来小程序这种应用形态消失的时候，`MorJS` 也会迎来它使命的终结亦或是换个方式重生，但在此之前，大家可以放心大胆地使用 😄 ，我们会一如既往的为社区提供高质量的服务，也非常欢迎志趣相投的小伙伴一起参与共建，让小程序研发能够每天幸福一点点。

**Reply by coalo** [reply]
请问是否还有未满的钉钉群可以加入呢，我现在使用mor遇到了一个无法解决的问题需要咨询一下大神们
